### ISSUE TITLE
[Help] Article miniature

### ISSUE THREAD

**Issue by iranzo** [Pelican-Elegant/elegant#550]
## Mandatory Step

- [ ] I am using latest production release of Elegant

## I need help with

According to https://elegant.oncrashreboot.com/search-engine-and-social-media-optimization, I can add featured_Image: to my yaml preamble and see the header for og:image defined, but I was somehow using that image also on the blog listing.

For example, let's say that I've an article about a gallery of pictures, I would like to be able to define there the first image in the gallery (even if manually) and have that shown in the article listing on frontpage or even search results

Is that a bad assumption and a feature request should be raised instead?

**Comment by talha131** [comment]
@iranzo I am somewhat confused.

>  and have that shown in the **article listing on frontpage** or even search results

If I understand correctly, you want the image to appear in the list of articles on the front page. I mean this place

![](https://i.imgur.com/j8jNeEo.png)

If so then we are talking about a major change in the look and feel of the theme. We can add it as an optional feature but it would require more than few changes in the "Recent Posts" section.

**Comment by iranzo** [comment]
Yup, exactly, similar to the author blurb which contains image, but for featured_image or something like that on articles.

Would that make sense? (of course as optional)

**Comment by talha131** [comment]
> Yup, exactly, similar to the author blurb which contains image, but for featured_image or something like that on articles.

We show the author's image in the article. If you want to show the image in the article list then

1. It will have to be optional
1. Small size images so that theme stays "minimum"

PR is welcomed.

**Comment by iranzo** [comment]
The image was proposed to the article list (as we already have the author one)

**Comment by talha131** [comment]
yeah, I understand. You want to show images here.

![](https://i.imgur.com/Sd3VBMD.png)

**Comment by iranzo** [comment]
There, and on search results and probably might make sense with a smaller image on the archives, etc
